content/en/serverless/installation/python.md

```diff
@@ -202,6 +202,55 @@ You need to subscribe the Datadog Forwarder Lambda function to each of your func
 [3]: https://docs.datadoghq.com/serverless/forwarder/
 [4]: https://docs.datadoghq.com/logs/guide/send-aws-services-logs-with-the-datadog-lambda-function/#collecting-logs-from-cloudwatch-log-group
 {{% /tab %}}
+{{% tab "Chalice" %}}
+
+### Update the Chalice project
+
+1. Set environment variables `DD_TRACE_ENABLED` and `DD_FLUSH_TO_LOG` to `"true"` in your `config.json`:
+    ```json
+    {
+      "version": "2.0",
+      "app_name": "hello-chalice",
+      "stages": {
+        "dev": {
+          "api_gateway_stage": "api",
+          "environment_variables": {
+            "DD_TRACE_ENABLED": "true",
+            "DD_FLUSH_TO_LOG": "true"
+          }
+        }
+      }
+    }
+    ```
+1. Add `datadog_lambda` to your `requirements.txt`.
+1. Register `datadog_lambda_wrapper` as a [middleware][1]:
```

Comment:
I think we should specify the file this is happening inside of, just like the previous two steps